docs/source/agents_list.rst

```diff
@@ -1,15 +1,14 @@
-# Standard Agents
+Standard Agents
+---------------
 
 ParlAI comes with a wide variety of standardized agents. Some of these are
 common baselines made available for quicker research; some can be made
 state-of-the-art with careful tuning and experimentation. Each of these contains
 a README with examples of usage and description of the agent.
 
-```eval_rst
 .. toctree::
    :maxdepth: 1
    :caption: List of Agents
    :glob:
 
    agent_refs/*
```

Comment:
Reading up on myst_parser, it seems like it's built to augment Markdown with all the features that rst has.

So it might be more straightforward if we keep all the files md, and use the myst syntax to replace eval_rst. What do you think?

For instance, toctree can be specified with:
```
```{toctree}
```

Reference: https://myst-parser.readthedocs.io/en/latest/using/faq.html

I'm not actually familiar with rst or myst, though, so feel free to ignore if you've already thought about this.

Reply:
Hm good point. I would like us to move to fully md somehow, but I was observing that myst didn’t have a way to trigger autodocs